CS.Security/Program.cs

```diff
@@ -153,7 +148,7 @@ private static async Task SeedRequiredData(WebApplication app)
         {
             using var scope = app.Services.CreateScope();
             var seedService = scope.ServiceProvider.GetRequiredService<DataSeeder>();
-            if (seedService == null) return;
+
```

Comment:
GetRequiredService either returns an instance of service, or throws an exception. So there is no need to check it for null reference